resolver/namerecord.go

```diff
@@ -32,7 +46,8 @@ type NameRecord struct {
 	Answer   []string
 	Ns       []string
 	Extra    []string
-	TTL      int64
+	// TODO: Name change in progress. Rename "TTL" field to "Expires" in Q1 2021.
+	TTL int64 `json:"Expires"`
 
```

Comment:
If we ever start writting queries for the NameRecord than https://github.com/safing/portbase/issues/75 will bite us! Why not rename the field right now? It's a cache database anyway...